fcrepo-webapp/pom.xml

```diff
@@ -365,6 +365,7 @@
               <value>classpath:/config/file-simple/repository.json</value>
             </systemProperty>
           </systemProperties>
+          <jvmArgs>${jetty.jvm.args}</jvmArgs>
```

Comment:
It doesn't directly.  I dropped that in there because helpful for debugging.  In the future I'll create a ticket and an appropriately named branch.